src/TodoItem.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+
+const TodoItem = ({ todo }) => (
+  <tr>
+    <td>{todo.id}</td>
+    <td>{todo.title}</td>
+    <td>{todo.completed ? 'finished' : 'not finished'}</td>
+    <td><User user={todo.user} /></td>
```

Comment:
```suggestion
    <User user={todo.user} />
```
а сам компонент пусть возвращает 
```
<td>${user.name}</td>
```
Так лучше читается, как мне кажется